src/highlight/HighlightCreator.tsx

```diff
@@ -7,5 +7,18 @@ type Props = {
 };
 
 export default function HighlightCreator({ className }: Props): JSX.Element {
+    React.useEffect(() => {
+        const textLayerEl = document.querySelector('.textLayer');
```

Comment:
This ties our implementation directly to pdfjs, which we've worked hard to avoid so far. Should this be managed by the Preview SDK, instead?